gatsby/src/components/Layout/Tables.scss

```diff
@@ -0,0 +1,72 @@
+@import '~@nice-digital/nds-core/scss/core.scss';
+
+// Taken from Niceorg Client
+
+.table {
+  width: 100%;
+  border-collapse: collapse;
+  th {
+    vertical-align: bottom;
+  }
+  td {
+    vertical-align: top;
+  }
+  td,
+  th {
+    text-align: left;
+    padding: rem($nds-spacing-small);
+    p {
+      margin-top: 0;
+    }
+  }
+  tr {
```

Comment:
Because we're in control of the CSS and the markup comes from the feed as raw markup, we only need the default table styling stuff, without success, error bordered etc.